estate/views/estate_menu_views.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <!-- TOP MENU -->
+    <menuitem 
+        id="estate_menu_root"
+        name="Estate"
+    />
+
+    <!-- HEADER: Advertisements -->
```

Comment:
You don't need these comments. If you don't find this way clear enough, you can nest your menuitems:
```
<menuitem id="coucou">
    <menuitem id="salut"/>
<menuitem/>
```

"salut" sera un sous-menuitem de "coucou"